doc/classes/Geometry2D.xml

```diff
@@ -125,8 +125,28 @@
 			<param index="2" name="from_b" type="Vector2" />
 			<param index="3" name="dir_b" type="Vector2" />
 			<description>
-				Checks if the two lines ([param from_a], [param dir_a]) and ([param from_b], [param dir_b]) intersect. If yes, return the point of intersection as [Vector2]. If no intersection takes place, returns [code]null[/code].
-				[b]Note:[/b] The lines are specified using direction vectors, not end points.
+				Checks if the two lines ([param from_a], [param dir_a]) and ([param from_b], [param dir_b]) intersect. Returns a [Vector2], or [code]null[/code] if the lines are parallel.
+				[code]from[/code] and [code]dir[/code] are **not** endpoints of a line segment or ray but the slope ([code]dir[/code]) and a known point ([code]from[/code]) on that line.
+				[codeblocks]
+				[gdscript]
+				var from_a = Vector2.ZERO
+				var dir_a = Vector2.RIGHT
+				var from_b = Vector2.DOWN    # y == 1 in 2D
```

Comment:
Why the additional spacing before the comment here? What does the comment mean, too? I'm not sure at a glance.

Reply:
I always forget that godot uses an inverted Y-axis in 2D, so I like having a reminder. We can clear this out if it's unnecessary though